src/components/RulesModal.vue

```diff
@@ -0,0 +1,112 @@
+                ○ If there are no more cards in the draw pile, shuffle the discard pile to create a
+                new draw pile. <br/>
+                ○ If there are no more cards in either the draw or the discard pile, move to the next
+                step.</p></li>
+              <li>Put a card to the left or right of your Decision card.<br/>
+                ○ If you cannot play a card, discard one card from your hand.</li>
+              <li>At the end of everyone’s turn, the Boolean coin is flipped and placed back next to
+                the deck.<br/>
+                ○ If the Boolean coin is “True” (Heads), the cards to the left-side of your Decision
+                card are ‘running’ to give your current total.<br/>
+                ○ If the Boolean coin is “False” (Tails), the cards to the right-side of your Decision
+                card are ‘running’ to give your current total.</li>
+            </ul>
+
+              <h4>Card Playing Rules</h4>
+              <ul>
+                <li><h5>Instruction Cards</h5>
+                <ul><li>Instruction cards are placed to either the left or right side of your Decision card.</li>
+                </ul></li>
+                <li><h5>Repetition Cards</h5>
+                <ul><li>Repetition cards are played on either an Instruction card or a Group card.</li>
+                <li>A variable Repetition card (R x ) can replace a R 2 , R 3 , R 4 , or R 5 on a player’s turn. The
+                  fixed Repetition card is then put in the discard pile.</li>
+                  <li>An Instruction card under a Repetition card can be grouped with another Instruction
+                    card. The Repetition card then applies to the Group card.</li>
+                  <li>A variable Repetition card (R x ) without a variable card has no effect.</li>
+                </ul></li>
+                <li><h5>Group Cards</h5>
+                <ul><li>A Group card replaces two Instruction cards that are on the same side of the
+                  Decision card and that add to the number on the Group card. For example, a Group-3
+                  card replaces both an Instruction-1 and an Instruction-2 card.</li>
+                <li>Group cards cannot be hacked.</li>
+                </ul></li>
+                <li><h5>Variable Cards</h5>
+                <ul><li>Variable cards can only be played on one Repetition-X card.</li>
+                <li>Variable cards can be replaced by a Variable card of a higher value. The replaced card
+                  is put in the discard pile. For example, a Variable-2 card can be replaced by a Variable-3
+                  card, and the Variable-2 is placed in the discard pile.</li>
+                </ul></li>
+              </ul>
+            </ul>
+            <h4>End of Game</h4>
+            <p>The game ends when the Boolean coin is flipped and you or your team’s ‘running’ program
+              totals 35 or more instructions.</p>
```

Comment:
We now have different score levels and the game stops immediately when someone reaches that total. Perhaps we should have "win" calculated on the coin flip. This would allow multiple players to potentially win (which might be good for kids).